modules/amd-creating-internal-registry.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
```

Comment:
@egallen @yevgeny-shnaidman Removed this module - is contained in the linked topic from AMD.